src/adapter/src/catalog.rs

```diff
@@ -533,6 +533,7 @@ impl Catalog {
             mz_catalog::durable::persist_backed_catalog_state(
                 persist_client,
                 environment_id.organization_id(),
+                &MetricsRegistry::new(),
```

Comment:
personally, I'd make this a parameter and fill this `&MetricsRegistry::new()` in the single caller (testdrive) but ymmv

Reply:
I think that sounds a lot cleaner, done.